docs/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 
 ## Unreleased
 ### Added
+* Added JsonFormat ([#950]).
```

Comment:
You also need to link `#950` to the pr link
like this:
`[#950]: https://github.com/amethyst/amethyst/pull/950`

Reply:
Ah i thought it would work like github references: #950. Fixed

Reply:
Thanks!